.github/workflows/release.yml

```
@@ -1,32 +1,94 @@
-name: Release
+name: Release Gem
 
-on:
-  push:
-    tags:
-      - '*'
+on: workflow_dispatch
 
 jobs:
   release:
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'voxpupuli'
+    if: github.repository == 'puppetlabs/beaker-puppet'
     steps:
       - uses: actions/checkout@v4
+
+      - name: Get Current Version
+        uses: actions/github-script@v7
+        id: cv
+        with:
+          script: |
+            const { data: response } = await github.rest.repos.getLatestRelease({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            console.log(`The latest release is ${response.tag_name}`)
+            return response.tag_name
+          result-encoding: string
+
+      - name: Get Next Version
+        id: nv
+        run: |
+          version=$(grep VERSION lib/beaker-puppet/version.rb |rev |cut -d "'" -f2 |rev)
```

we can shorten this and use less pipes:
```suggestion
          version=$(awk '/VERSION/ {print $3}' lib/beaker-puppet/version.rb | tr --delete \')
```